pkg/query-service/app/preferences/map.go

```diff
@@ -11,4 +11,14 @@ var preferenceMap = map[string]Preference{
 		IsDiscreteValues: true,
 		AllowedScopes:    []string{"org"},
 	},
+	"THIRD_ONBOARDING": {
```

Comment:
Is it used somewhere ? 

also why is the name `THIRD_ONBOARDING` instead of `THIRD_PATY_ONBOARDING` ?